less/navigationBar.less

```diff
@@ -660,19 +660,24 @@
     z-index: @zindexUrlbarNotLegend;
   }
 
-  input:focus + legend:before {
+  legend:before {
     content: ' ';
     position: absolute;
     background: @navigationBarBackgroundActive;
-    border-radius: 4px;
-    box-shadow: 0 0 1px @focusUrlbarOutline, inset 0 0 2px @focusUrlbarOutline, inset 0 1px 8px rgba(0, 137, 255, 0.1);
+    border-radius: 0 4px 4px 0;
     color: #333;
+    box-shadow: inset 0 0 0 1px #bbb, inset 0 0 0 3px rgba(55, 169, 253, 0.4);
```

Comment:
Let's replace `#bbb` and `rgba(55, 169, 253, 0.4)` with variables and add them to variables.less and global.js for future refactoring work.